### ISSUE TITLE
Issue on page /docs/02-structure.html

### ISSUE THREAD

**Issue by jccaicedo** [cytomining/DeepProfiler-handbook#6]
* Update the example data section (2.2)
* Bring metadata to a separate chapter and expand that chapter with locations (Metadata and single-cell locations)
* Bring the cell masks to a separate chapter and expand with images information (Images and cell masks)
* Figure 2 needs to be updated (pretrained goes away and we need to bring outlines).

**Comment by Arkkienkeli** [comment]
Hi @rsenft1 @yhan8, I have updated the example data information and created the new chapters with the text from chapter 2. Feel free to reorganize \ rewrite those new pages. Thank you!

**Comment by yhan8** [comment]
I just updated the [03-images.md] section.

**Comment by jccaicedo** [comment]
@Arkkienkeli we need an example `config.json` file here to run the very first `setup` command. Where can we find this initial `config.json` file?

Alternatively, we can make DeepProfiler ignore the `config.json` parameter for the `setup` command given that it is not really required.

Either way, please, make sure the instructions point to an existing `config.json` file or that the instructions remove the parameter if no longer needed.

**Comment by jccaicedo** [comment]
I think the best way to go is to add a section here in Chapter 2 called **Configuration File**, give a brief introduction of what this is (without explaining any options), and indicate that it is necessary for running all commands of DeepProfiler. Also, provide a link for a configuration file (or a procedure to download it or obtain it).

**Comment by jccaicedo** [comment]
The example configuration files are also discussed in issue #7

**Comment by Arkkienkeli** [comment]
> @Arkkienkeli we need an example `config.json` file here to run the very first `setup` command. Where can we find this initial `config.json` file?
> 
> Alternatively, we can make DeepProfiler ignore the `config.json` parameter for the `setup` command given that it is not really required.
> 
> Either way, please, make sure the instructions point to an existing `config.json` file or that the instructions remove the parameter if no longer needed.

I don't think that config is needed to run prepare command, I would prefer to fix that in the code and then reflect this change in the documentation. 

> I think the best way to go is to add a section here in Chapter 2 called **Configuration File**, give a brief introduction of what this is (without explaining any options), and indicate that it is necessary for running all commands of DeepProfiler. Also, provide a link for a configuration file (or a procedure to download it or obtain it).

If it won't be needed to run `prepare`, then we don't need to swap sections.

**Comment by Arkkienkeli** [comment]
I have updated DeepProfiler, so `setup` does not need config anymore.

**Comment by Arkkienkeli** [comment]
We need to do is just remove `--config=config.json` from the command in section 2.

**Comment by rsenft1** [comment]
✅ Removed `--config=config.json` from the command in section #02.

Working on metadata/single cell locations now...

**Comment by rsenft1** [comment]
✅ The figure has been updated and looks like this: 
<img width="233" alt="image" src="https://user-images.githubusercontent.com/28116530/192605847-d5fc4032-7969-4105-ab55-3dd75a83bd20.png">

✅ Locations description added

✅ Index.csv file is linked (it goes to the page on GitHub, I figured this would be better than linking to raw?)

I do have a few questions (posted to [slack](https://broadinstitute.slack.com/archives/CPSCBNTNJ/p1664311299362579)) But otherwise I believe my part here is done. Please comment if anything doesn't make sense with what I've written!

**Comment by jccaicedo** [comment]
Wonderful! This looks great!
